bin/awslocal

```diff
@@ -101,19 +101,15 @@ def prepare_environment():
     env_dict = os.environ.copy()
     env_dict['PYTHONWARNINGS'] = os.environ.get(
         'PYTHONWARNINGS', 'ignore:Unverified HTTPS request')
-    if os.environ.get('DEFAULT_REGION'):
```

Comment:
`DEFAULT_REGION` is deprecated and non-functional in LocalStack core since v1

Removing here too